src/SCREENS.ts

```diff
@@ -24,6 +24,7 @@ export default {
     SIGN_IN_WITH_APPLE_DESKTOP: 'AppleSignInDesktop',
     SIGN_IN_WITH_GOOGLE_DESKTOP: 'GoogleSignInDesktop',
     DESKTOP_SIGN_IN_REDIRECT: 'DesktopSignInRedirect',
+    MOBILE_TEMP_SCREEN: 'MobilePlaceHolder',
```

Comment:
@b4s36t4 I'm not familiar with the navigation system but why do we need this blocking component as a new screen?

isn't the new component just displayed when the screen size is small in another component?

Reply:
This component shouldn't be registered as a new screen